src/SubjectsHandler.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Queries for all subjects in the current queryEngine source
+ */
+export const SELECT_ALL_SUBJECTS = 'SELECT distinct ?thing WHERE { ?thing ?pred ?type }';
```

Comment:
Let's just inline (only used once).
Also, consistent caps in `DISTINCT`, and perhaps just `?s ?p ?o` or `?subject ?predicate ?object`.

Reply:
And definitely not export, this is an internal matter.